<launch>

    <arg default="20" name="rate"></arg>
    <arg default="0.5" name="accelNoise"></arg>
    <arg default="0" name="magAngleOffset"></arg>
    <arg default="tag6752" name="targetDeviceId"></arg>
    <arg default="/gtec/toa/ranging/filtered" name="rangingTopic"></arg>
    <arg default="/gtec/gazebo/uwb/anchors/0" name="anchorsTopic"></arg>
    <arg default="0" name="tagId"></arg>
    <arg default="ALGORITHM_KF_UWB" name="algorithm"></arg>
    <node name="kfpos_uwb$(arg targetDeviceId)" output="screen" pkg="gtec_roskfpos" type="kfpos" launch-prefix="">
        <param name="rate" type="int" value="$(arg rate)" />
        <param name="accelNoise" type="double" value="$(arg accelNoise)"></param>
        <param name="magAngleOffset" type="double" value="$(arg magAngleOffset)"></param>
        <param name="targetDeviceId" type="string" value="$(arg targetDeviceId)"></param>
        <param name="rangingTopic" type="string" value="$(arg rangingTopic)"></param>
        <param name="anchorsTopic" type="string" value="$(arg anchorsTopic)"></param>
        <param name="tagId" type="string" value="$(arg tagId)"></param>
        <param name="algorithm" type="string" value="$(arg algorithm)"></param>
    </node>
</launch>
